stetho/src/main/java/com/facebook/stetho/common/android/ResourceIdUtil.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.common.android;
+
+import android.content.res.Resources;
+import com.facebook.stetho.common.LogUtil;
+
+import javax.annotation.Nonnull;
+
+public class ResourceIdUtil {
+  private ResourceIdUtil() {
+  }
+
+  @Nonnull
+  public static String getIdStringQuietly(Resources r, int resourceId) {
+    try {
+      return getIdString(r, resourceId);
+    } catch (Resources.NotFoundException e) {
+      String idString = "#" + Integer.toHexString(resourceId);
+      LogUtil.w("Unknown identifier encountered: " + idString);
```

Comment:
if I saw this in logcat I'd have no idea where it came from ... "Unknown identifier encountered" is very generic! let's tag it with class/method name or something